# Semantic-Web-for-Genomics
* **Objective**: Develop Computational Tools for a Systems Approach to Genomics Analysis
* **Background**: We carry 3 billion base pairs of nucleotides in our nuclear DNA sequence. Only a tiny part of this genome sequence represents protein coding genes, with much of the remainder sequence involved in gene regulation and consequent expressed levels of these proteins. Regulation is a complex process dependent on the tissue type and our individual profile (dietary, lifestyle and environmental factors).   Generally, genomic origins of rare diseases is better understood than for common and complex diseases such as cancer, heart disease, diabetes, and Alzheimer’s, as well as the processes of aging and cell death.   Potential heritability of traits (such as height) and disease risk (as with, say, heart attack) has been determined by twin, family, and ethnic studies, to be around 40 to 60 %, or more. However, heritability as explained by known gene-sequence variants is less than 10 %. Thus the new challenge for bioinformatics is to integrate, and gain insight from, the massive amounts of knowledge and data acquired across species, genes, proteins, and biological processes, as well as clinical studies.  The eventual goal is to provide personalized medicine. The first step, started a decade ago, has been the development and applications of genome and disease ontologies. The second step, started more recently, relates to building genetic architecture and biological pathway models for overlapping and specific aspects of diseases. We are offering a course in spring '19 that will address the first step in more detail, with a brief introduction on the latter at semester end. 
* **Text Book**: :  A free download: Dessimoz, C., and Skunca, N., [The Gene Ontology Handbook](https://link.springer.com/content/pdf/10.1007%2F978-1-4939-3743-1.pdf), publishers: Springer Open and Humana Press, 2017. There are many online resources for the genome-related ontologies (GO and BioPAX) and data mining/ inferencing standards and tools to be used: OWL, Protégé, AmiGO 2, and REVIGO.
* **Documentation Here**: Our goal here is to document all relevant information for this course, so it is easily accessible.  This is a bioinformatics application course. It represents an **intermediate step** in the evolution of the field. The **first step** (['Bioinformatics'](https://en.wikipedia.org/wiki/Bioinformatics)) was the application of data science and analytics techniques (pattern analysis, data mining, and machine intelligence algorithms and visualization) for DNA/RNA sequence alignment, gene discovery, and genome assembly

It uses a small standardized vocabulary (the GO ontology) to help lower the barrier for non-biologists to make sense of the Big Data in open source genome and disease repositories. Engineering, nursing, biology, and IT professionals can then apply their perspectives / training to develop innovative approaches and applications. 
