This assignment is on GWAS-Catalog based extension of your gene/gene product set. Description follows:
For this assignment, use GWAS catalog to broaden/narrow/focus as you wish (and is appropriate) to identify a set of genes/gene products based on different factors (disease conditions, environment factors, biomarkers, measurements, attributes, etc.,). You can use multiple key words and sort at the website. Or, you can download and save in Excel, and then sort. Other key factors to consider as potential modulators of disease risk:  aging, inflammation, stress, exercise, dietary habits, and ethnicity/geographical variations. Consider as many or as few. Discuss appropriateness of your choices. 

Note: There is no correct or required answer. What is needed is a reasoned application focus: help patients with certain complex disease. Leave your imprint on the report! For example, my focus is ECM (extracellular matrix) and their impact on nonlinear artery properties. The focus  on the genomic side has come after 30 years of exploration, but was generally defined and well funded in the bioengineering area. You probably have some preference due to personal, familial, or professional health concerns. Use that to advantage. But, as relevant to CVD. 

Continue with your chosen topic.  Write a 5 page report, as a regular conference paper (with sections as follows: Title, authors, abstract, background, methods, results in a tabulated format, discussion, conclusions, references, and appendices).

Upload  a PDF document to Canvas. Also, either upload an editable version (as with MS Word) to Canvas or give a link to an online shared document (as with Google Doc) and share it with shankar@fau.edu.  

Earlier comments before the assignment was finalized: 

It will involve the use of the GWAS catalog to find the larger universe of potential (or putative) loci for the specific disease/risk factor (plus similar diseases). Some of it probably has been incorporated into GAF. GO, as a consequence, may reflect some of these advances. But, do not start with GO. 

One student, because of his prior work with the GO ontology, was able to undertake this. One obtains a much larger universe of GO terms. However, there are limitations with this approach. A majority of the annotated additions are of electronic in nature, based on algorthmic deductions, not experimental/clinical research studies. About 10% of such annotations seem to have been validated in the ensuing 5 to 10 years. So, while it is a good comprehensive list, by no means, it can be considered directly relevant or valid. Two other considerations: The GO terms, especially at the higher levels, are not tissue specific and are more generic. So, it is necessary to prune the list of gene terms and gene products based on tissue and disease specificity. Also, research cannot be trusted unless and until it has been replicated. So, further trimming of the list based on if the study has been validated by others is essential. Also, note that the GAF may not be up-to-date vis-a-vis the list at GWAS catalog.

Assignment Description:

