Report:

This is tailored to the individual student. The goal is to write a short 4 to 6-page report that is publishable.

One option is to focus on what is useful to the scientific reader who is in the genomics field and could use some information to further his/her own goals. Generic enough, not too specific to your topic of focus. They may not be focusing on your specific topic; so, details of your work may not be beneficial.  However, you can use that as an example, and document all the topic related details at our GitHub repository, providing a reference to that in your report. Such a report would likely be process-based, as with Adam (providing a process that can be automated for their searches), or Meric (Java-based method to identify the ‘lowest common denominator’ node). As they progress this summer (if they wish to), their work could lead to a paper on genome-to-phenome (G-to-P) mapping, with focus on their topic. That would be a second paper!  I think Adam and Meric can target the Las Vegas conference mentioned in the Canvas announcements (deadline to submit: 4/26).

A second option is to go the opposite way. First focus on a more specific G-to-P mapping (and extract a process paper later on). Note: we are doing actually P-to-G mapping. This paper would encompass all the stages of innovation in this field (pre-GWAS, post-GWAS, and now NGS, during 2015-2019), to the extent you can. NGS has used the GWAS results as the starting point for further focused studies (as with WGS – whole genome sequencing; revisited WES – whole exome sequencing, eQTL – expressed quantitative trait loci, and epigenomics – methylation and histone modifications).  Such studies can be searched for through Google Scholar. The recent slide presentation in the class will help put this in perspective (ref: Australian MS study). They used a second focused study with fewer SNPs. Chapters 5 and 22 on ADR (adverse drug reaction) and CAD (coronary artery disease) of “Cardiovascular Genetics and Genomics” would help identify certain orphan tag SNPs and gene products. I showed how to take such orphan SNPs forward to find gene products via SNPedia (in the class of 4/19/19). If you actually go forward and identify the unknown gene, then that will be another (third type) paper! That will require bioinformatics training.

So, where to stop for the second type of paper? It depends on the conference. However, it is good to mention these and take at least one or two such tag SNPs/gene products forward so they can be included in GO searches. Indicate in the discussion section that further work is planned.

In a typical paper, the what is in the background, how is in methods, examples will be in results, and the why and what else in the discussion section.  The abstract will extract one or two lines from each of these sections and integrate them. The abstract and conclusion sections are read first by the readers. Use the conclusion section to bring out some interesting aspects of your work. This will be followed by the reference section.

Upload a Word or PDF document (preferably a Word document).  In addition, a link to a shared Google Doc will help continue the collaboration. If so, please give the link here. Do not forget to give me access.

I may add more comments later. 

Good luck!

Dr. Shankar
