Notes from: Epigenetics - A Graphic Guide, 2017, by Cath Ennis and Oliver Pugh. It is a light-hearted and comic strip-oriented introduction to the field. 

"about how the genes we inherit from our parents are controlled, and how they interact with our environment: how our genes make us, well, us."

Almost every cell in our body contains its own copy of our unique version of the human genome. Mature red blood cells and cornified cells in the skin, hair, and nails contain no nucleus. The longest human Chr contains ~2600 protein coding genes; the smallest, just 140. 

The double helix first wraps around a cluster of 8 small proteins called **histones**, which bind very tightly to DNA. Each individual unit of 8 histone (four types) proteins plus 146 bases of DNA == '**nucleosome**'. The DNA - nucleosome assembly looks like **beads on a string**. A fifth type of histone protein attaches to the linker DNA between nucleosomes. This compacts the assembly into a thicker strand. Stretched out end-to-end, DNA will be ~ 1.8 m long (5'). DNA + histones + scaffold proteins + other proteins + RNAs == **Chromatin**. 

1962: Gurdon artifically reversed cell differentiation of a mature fully differentiated cell. This proved that such adult cells retail all the genetic material needed to produce every cell in the body. This disproved the 'use it or lose it' concept. Tijan in 1978 discovered **Transcription Factor** proteins, which help regulate gene activation. TFs bind specific DNA sequences near the genes, and interact with the transcription machinery. The combination of proteins so bound determine whether transcription takes place. Additional mechanism, **imprinting**, may also control certain genes (75 known in the human genome; 150 in the mouse genome). From Wikipedia,  " It is an epigenetic process that involves DNA methylation and histone methylation without altering the genetic sequence. These epigenetic marks are established ("imprinted") in the germline (sperm or egg cells) of the parents and are maintained through mitotic cell divisions in the somatic cells of an organism."

**Chromatin remodeling**: Allis in mid-1990s discovered chemical changes to histone proteins, changes that correlate with gene transcription. Addition of **acetyl** group (a small molecule) to histones led to more **actively transcribed** genes. His group also found other modifications that led to **silent** genes. **Physical configuration** of the chromatin structure correlates well with gene transcription (the denser, the less active). Bird in 1970s found that methylation of the genome is more specific to the gene regions. **Chromatin modifications** bridge the gap between the DNA seq and the phenotype. Now the *cause and effect part*: intuitively, denser structures physically block the transcription machinery from reaching the DNA. Chemically, US and German teams in 1980s found that unmethylated DNA is more transcriptionally active; Allis in 1990s showed acetylation addition to histones activates gene transcription in some cases. 

Modern understanding: "Epigenetic info is a highlighted and annotated version of the text." There are thousands of epigenetic *highlighters, erasers, and decoders* (RNAs and proteins). Smallest known has only four atoms (the methyl group - C with 3 Hs- is attached to DNA's C bases by a DNA methyltransferase - causes gene silencing - 'blacks it out'); RNA strands 19 bases long specify where each highlighting should occur. 

Frommer and Clark developed a method to reveal exactly which C bases are methylated. **Bisulphite sequencing** converts unmethylated C bases into Ts; methylated Cs retain their C base. This is the standard way of mapping gthe epigenetic landscapes of different types of cells. 

Most such bases are adjacent to G bases. Many active genes have CpG islands near the transcription start sites. Rare in human genome; 70% of such islands have Cs methylated. Evolutionary: *Repititive DNA* is problematic - can move to new locations, can cause transcription and replication to slip and stumble, causing mutations that can lead to cancer etc. It is possible that DNA methylation originally evolved to shut such sequences down; useful funcitons evolved later. 

An aside: A 2011 genome-wide study of [methylation patterns](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3044686/) and gene expressions in human and Chimpanzee tissues: Compared liver, heart, and kidney tissues. They studies 7723 genes. "inter-species differences in promoter methylation might underlie as much as 12%–18% of differences in gene expression levels between humans and chimpanzees." Some sequencing method details: Illumina HumanMethylation27 DNA Analysis BeadChip assay, which provides  methylation levels at 27,578 CpG-loci near transcription start sites. The 50 bp probes on the Illumina array are designed to interrogate human samples; so,limited to probes that were a perfect sequence match to the chimpanzee genome. In addition, they only used probes that were associated with genes for which they had expression measurements across the three tissues. They  thus retained 10,575 CpG site probes in the putative promoter regions of 7,723 genes. At each probe, DNA methylation levels were estimated using the Illumina-recommended β values, which are essentially estimates of the proportion of methylated DNA at each CpG site.

DNA methylation is the best known example of a mitotically heritable epigenetic modification. During active demethylation, the methyl groups that need to be removed are tagged with oxygen atomes. "*Eraser*" proteins called Tet bind to such tagged methyl groups and snip them off the DNA. "*Highlighter*": methyl group, etc. "DNA methylation is absolutely essential for the development and health of the organisms." 

Histone modifications: Histone proteins can also be tagged wigth methyl groups, as DNA is, typically to histone 'tails.' These patterns change more often and more quickly relative to DNA methylation patterns. May be associated with short-tgerm fluctuations. *Comment*: Is there a role in adaptation and optimization of the system in normal tissues? There are *decoder* proteins that bind to histones and regulate the acrtivity of nearby genes. 

It is relatively easy to id the precise location of each methylated C base in the DNA. However, each nucleosome is associated with 150 bases of DNA  + 80-base linker sequence. **Chip-Seq** (Chromatic Immunoprecipitation Sequencing) determines histone modifications. DNA methylation is always associated with gene silencing. However, histone modification can either silence or activate the gene. Different types of **histone methylation** id parts of the genome with different characteristics: active/silent genes; DNA to be repaired; helper DNAs that control near or remote genes. Acetyl and phosphate, also small molecules, can also attach to histone tails. **Histone acetylation** was the first histone modification to be discovered. It is associated with gene activation. **Histone phosphorlyation** is associated with DNA repair. 

Strome in **2014** showed that methylation mark can be passed on through cell divisions. So, some of the epigenetics can be inherited...






